src/YNCursesUI.cc

```diff
@@ -423,3 +424,24 @@ void YNCursesUI::setConsoleFont( const std::string & console_magic,
     }
 }
 
+
+YWidget * YNCursesUI::askSendWidgetID()
+{
+    std::string id = NCPopupTextEntry::askForText( wpos( 0, 0 ),
+                                                   "Widget ID:", // label
+                                                   "" ); // initial text
+
+    if ( ! id.empty() )
+    {
+        try
+        {
+            return sendWidgetID( id );
+        }
+        catch ( YUIWidgetNotFoundException & ex )
+        {
+            YUI_CAUGHT( ex );
```

Comment:
Um, it should display an error popup, like the Qt UI: https://github.com/libyui/libyui-qt/pull/74/files/bb0fa0426b545c8740007bba8c14a7d1cfbd91d9#diff-0ffb2fa84e56876f016657d53238482eR643

Reply:
I was considering that, but we don't seem to have a simple pre-made function for that, unlike in Qt.

Reply:
OK, displaying a popup is not trivial in NCurses UI, so let's skip it here.